core/pkg/server/handler_test.go

```diff
@@ -715,7 +715,7 @@ func TestHandlePartialHistory(t *testing.T) {
 			fwdChan := make(chan *service.Record, server.BufferSize)
 			outChan := make(chan *service.Result, server.BufferSize)
 
-			makeHandler(context.Background(), inChan, fwdChan, outChan)
+			makeHandler(inChan, fwdChan, outChan, "")
```

Comment:
nit add comment after the empty string /*...*/

Reply:
done